pkg/controller/logstash/initcontainer.go

```diff
@@ -32,7 +32,7 @@ mount_path=` + volume.InitContainerConfigVolumeMountPath + `
 
 cp -f /usr/share/logstash/config/*.* "$mount_path"
 
-ln -sf ` + volume.InternalConfigVolumeMountPath + `/` + ConfigFileName + ` $mount_path
+cp ` + volume.InternalConfigVolumeMountPath + `/` + ConfigFileName + ` $mount_path
```

Comment:
Let's add a comment explaining that Logstash rewrites the configuration file in the presence of environment variable and that that's why we have to copy it.